test/data/examples/enrollmentrequest.yaml

```
@@ -1,34 +1,14 @@
-    BAwwCgYIKwYBBQUHAwIwDAYDVR0TAQH/BAIwADArBgNVHSMEJDAigCANii2X78Cy
-    RcU4ryE6gW7EDWhb8AundrVmJ3wQyrPg7jAKBggqhkjOPQQDAgNHADBEAiBHvTby
-    aI2MfLGq4dZKfSxEePxWwmPFaPmN1dfw+53vdgIgX7SNGvrdB+CQBvrfJFKoDu+o
-    O6WEM3Wl62xKjRnCFDs=
-    -----END CERTIFICATE-----
-  conditions:
-    - lastTransitionTime: "2023-10-10T17:10:32Z"
-      message: Approved by auto approver
-      reason: AutoApproved
-      status: "True"
-      type: Approved
+apiVersion: api.flightctl.io/v1alpha1  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)